feature/qos/ate_tests/two_sp_queue_traffic_test/two_sp_queue_traffic_test.go

```diff
@@ -1197,38 +1197,65 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	dp3 := dut.Port(t, "port3")
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
+
+	type qosVals struct {
+		be0, be1, af1, af2, af3, af4, nc1 string
+	}
+
+	qos := qosVals{
+		be0: "BE0",
+		be1: "BE1",
+		af1: "AF1",
+		af2: "AF2",
+		af3: "AF3",
+		af4: "AF4",
+		nc1: "NC1",
+	}
+
+	if dut.Vendor() == ondatra.JUNIPER {
```

Comment:
Where is the deviation that tracks the need for this change?

Reply:
This is not a deviation. it is a vendor specific queue name, No test expectation is modified. it is already approved 
https://github.com/openconfig/featureprofiles/blob/main/feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go

Reply:
Ok, I got confirmation that we are going to punt on this non-uniformity for now. I wish an issue had been created to arrive at a more uniform solution (such a helper naming library), but it hasn't. I'll track it elsewhere internally.